tests/scripts/analyze_outcomes.py

```diff
@@ -280,7 +280,260 @@ def do_analyze_driver_vs_reference(outcome_file, args):
+                    'PBE Decrypt, pad = 7 (OK)',
+                    'PBE Decrypt, pad = 8 (Invalid output size)',
+                    'PBE Decrypt, pad = 8 (OK)',
+                    'PBE Encrypt, pad = 7 (OK)',
+                    'PBE Encrypt, pad = 8 (Invalid output size)',
+                    'PBE Encrypt, pad = 8 (OK)',
+                ],
+                # Following tests depends on PCKS7
+                'test_suite_pkcs5': [
+                    'PBES2 Decrypt (Invalid output size)',
+                    'PBES2 Decrypt (Invalid padding & PKCS7 padding enabled)',
+                    'PBES2 Decrypt (KDF != PBKDF2)',
+                    'PBES2 Decrypt (OK)',
+                    'PBES2 Decrypt (OK, PBKDF2 params explicit keylen)',
+                    'PBES2 Decrypt (OK, PBKDF2 params explicit prf_alg)',
+                    'PBES2 Decrypt (bad KDF AlgId: not a sequence)',
+                    'PBES2 Decrypt (bad KDF AlgId: overlong)',
+                    'PBES2 Decrypt (bad PBKDF2 params explicit keylen: overlong)',
+                    'PBES2 Decrypt (bad PBKDF2 params iter: not an int)',
+                    'PBES2 Decrypt (bad PBKDF2 params iter: overlong)',
+                    'PBES2 Decrypt (bad PBKDF2 params salt: not an octet string)',
+                    'PBES2 Decrypt (bad PBKDF2 params salt: overlong)',
+                    'PBES2 Decrypt (bad PBKDF2 params: not a sequence)',
+                    'PBES2 Decrypt (bad PBKDF2 params: overlong)',
+                    'PBES2 Decrypt (bad enc_scheme_alg params: len != iv_len)',
+                    'PBES2 Decrypt (bad enc_scheme_alg params: not an octet string)',
+                    'PBES2 Decrypt (bad enc_scheme_alg params: overlong)',
+                    'PBES2 Decrypt (bad enc_scheme_alg: not a sequence)',
+                    'PBES2 Decrypt (bad enc_scheme_alg: overlong)',
+                    'PBES2 Decrypt (bad enc_scheme_alg: unknown oid)',
+                    'PBES2 Decrypt (bad iter value)',
+                    'PBES2 Decrypt (bad params tag)',
+                    'PBES2 Decrypt (bad password)',
+                    'PBES2 Decrypt (bad, PBKDF2 params explicit prf_alg != HMAC-SHA*)',
+                    'PBES2 Decrypt (bad, PBKDF2 params explicit prf_alg not a sequence)',
+                    'PBES2 Decrypt (bad, PBKDF2 params explicit prf_alg overlong)',
+                    'PBES2 Decrypt (bad, PBKDF2 params extra data)',
+                    'PBES2 Encrypt, pad=6 (OK)',
+                    'PBES2 Encrypt, pad=8 (Invalid output size)',
+                    'PBES2 Encrypt, pad=8 (OK)',
+                ],
+                # Following tests depends on DES
+                'test_suite_pkparse': [
+                    'Key ASN1 (Encrypted key PKCS12, trailing garbage data)',
+                    'Key ASN1 (Encrypted key PKCS5, trailing garbage data)',
+                    'Parse RSA Key #20 (PKCS#8 encrypted SHA1-3DES)',
```

Comment:
Same: things would be much more readable & maintainable if we could ignore by pattern, here `PKCS#8 encrypted` for example.